spec/models/test_track/remote/split_detail_spec.rb

```diff
@@ -27,6 +37,12 @@
       expect(subject.owner).to eq("fake owner")
       expect(subject.location).to eq("fake activity")
       expect(subject.platform).to eq("mobile")
+
+      expect(subject.variant_details.size).to eq 2
+      expect(subject.variant_details.first[:name]).to eq("fake first variant detail")
+      expect(subject.variant_details.first[:description]).to eq("There are FAQ links in a sidebar")
+      expect(subject.variant_details.last[:name]).to eq("fake second variant detail")
+      expect(subject.variant_details.last[:description]).to eq("There are FAQ links in the default footer")
```

Comment:
that happens below in the `with_test_track_enabled` block. 

these values are the default `fake_instance_attributes` from the Her model.